src/azure-cli-core/azure/cli/core/telemetry_upload.py

```diff
@@ -38,7 +38,7 @@ def send(self, data_to_send):
         request = HTTPClient.Request(self._service_endpoint_uri, bytearray(request_payload, 'utf-8'),
                                      {'Accept': 'application/json', 'Content-Type': 'application/json; charset=utf-8'})
         try:
-            response = HTTPClient.urlopen(request)
+            response = HTTPClient.urlopen(request, timeout=10)
```

Comment:
Is this timeout in seconds?  What happens to the data if we time out?  Is it lost?  Do we hold on to it and try to send it again with the next bundle?

Reply:
1. In seconds
2. Data lost (its telemetry data)
3. We don't retry

YOLO